spec/entry_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'entry'
+
+describe Entry do
+
+  let(:entry) { described_class.new }
```

Comment:
Could there be different sets of tests that use different of `let`s, depending on their setup needs?